package.json

```diff
@@ -29,7 +30,7 @@
     "minimatch": "^3.0.4"
   },
   "peerDependencies": {
-    "babel-eslint": "^10",
+    "@babel/eslint-parser": "^7",
```

Comment:
qq, should we also specify `@babel/core` as a peer dependency? The reason I ask is because it is a peerDependency [here](https://github.com/babel/babel/blob/main/eslint/babel-eslint-parser/package.json#L28-L31)

Reply:
No, I don't think so. This package depends on `@babel/eslint-parser` not on `@babel/core`. The fact that `@babel/eslint-parser` depends on `@babel/core` is an internal detail.

Reply:
We should remove `@babel/core` from the list of packages in the "installation" section.

Reply:
Never mind, I see that `@babel/core` is a peer dep of `@babel/eslint-parser`...